### PR TITLE
R115 blog post

### PR DESCRIPTION
**Pull request by benjben**

### CODE REVIEW

File: _posts/2019-07-18-snowplow-r115-sigiriya-eer/index.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+Still in an effort to improve the reliability of `EmrEtlRunner`, an update has been made so that `EmrEtlRunner` fails if an EMR step can't be successfully submitted to a transient EMR cluster.
+
+This was already the case for a standard EMR cluster.
+
+<h2 id="emp">2. Event Manifest Populator</h2>
+
+To solve the "cold start" problem for cross-batch deduplication in the RDB Shredder, we had developed the [Event Manifest Populator](https://github.com/snowplow/snowplow/tree/master/5-data-modeling/event-manifest-populator).
+
+It was developped to use events emitted by `spark-enrich` as input, and we now added the possibility to also read events emitted by `stream-enrich`.
```
Comment (goodits) [inline _posts/2019-07-18-snowplow-r115-sigiriya-eer/index.md]: Typo and change in wording. Do check my suggestion for accuracy!
```suggestion
It was developed to use events emitted by `spark-enrich` as the input, but we have now added the possibility to also read events emitted by `stream-enrich`.
```

Comment (goodits) [inline _posts/2019-07-18-snowplow-r115-sigiriya-eer/index.md]: Is the Event Manifest Popular something that _was_ added and is now being updated? Or is it new in this release? It's not 100% clear.

Comment (benjben) [reply _posts/2019-07-18-snowplow-r115-sigiriya-eer/index.md]: It was added and is now being updated. I will update to make it more clear.

File: _posts/2019-07-18-snowplow-r115-sigiriya-eer/index.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+<h3>1.2. Step failure on transient EMR cluster</h3>
+
+Still in an effort to improve the reliability of `EmrEtlRunner`, an update has been made so that `EmrEtlRunner` fails if an EMR step can't be successfully submitted to a transient EMR cluster.
+
+This was already the case for a standard EMR cluster.
+
+<h2 id="emp">2. Event Manifest Populator</h2>
+
+To solve the "cold start" problem for cross-batch deduplication in the RDB Shredder, we had developed the [Event Manifest Populator](https://github.com/snowplow/snowplow/tree/master/5-data-modeling/event-manifest-populator).
```
Comment (benjben) [inline _posts/2019-07-18-snowplow-r115-sigiriya-eer/index.md]:
```suggestion
To solve the "cold start" problem for cross-batch deduplication in the RDB Shredder, in R88 we had introduced the [Event Manifest Populator](https://github.com/snowplow/snowplow/tree/master/5-data-modeling/event-manifest-populator).
```

Would it be better @goodits ?

**Comment by yalisassoon** [comment]
LGTM @benjben - short and sweet :smile:

**Pull request merged**